stripe/src/main/java/com/stripe/android/model/AccountParams.java

```diff
@@ -0,0 +1,79 @@
+package com.stripe.android.model;
+
+import android.support.annotation.NonNull;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.stripe.android.StripeNetworkUtils.removeNullAndEmptyParams;
+
+/**
+ * Represents a grouping of parameters needed to create a Token for a Connect account on the server.
+ */
+public class AccountParams {
+
+    static final String API_PARAM_LEGAL_ENTITY = "legal_entity";
+    static final String API_TOS_SHOWN_AND_ACCEPTED = "tos_shown_and_accepted";
+    private Boolean mTosShownAndAccepted;
+    private Map<String, Object> mLegalEntity;
+
+    /**
+     * @param tosShownAndAccepted indicates that the platform showed the user the appropriate text
+     *                            and links to Stripe's terms of service. Tokens will only generated
+     *                            when this is true.
+     * @param legalEntity map that specifies the legal entity for which the connect account is being
+     *                    created. Can contain any of the field specified by legal_entity is the API
```

Comment:
typos: "any of the **fields** specified by legal_entity **in** the API"

Same typos below